### ISSUE TITLE
Help needed for ImageNet dataset that is already prepared as tf-records

### ISSUE THREAD

**Issue by vishal-keshav** [tensorflow/datasets#373]
I am looking for some help regarding **how to use the Imagenet dataset that has been already downloaded and processed as tfrecords files** by using the script  [download_and_convert_imagenet.sh](https://github.com/tensorflow/models/blob/master/research/slim/datasets/download_and_convert_imagenet.sh) present in the models/research/slim directory of tensorflow.

Tfrecords files that are generated looks like this:
```
train-00000-of-01024.tfrecords
train-00001-of-01024.tfrecords
.....
train-00821-of-01024.tfrecords
```
in train folder.

```
validation-00000-of-00128.tfrecords
validation-00001-of-00128.tfrecords
.....
validation-00127-of-00128.tfrecords
```
in validation folder

The tensorflow_dataset API for Imagenet requires us to specify the directory for these two files
`ILSVRC2012_img_train.tar` and `ILSVRC2012_img_val.tar` which I don't have.

I don't want to download ImageNet dataset again, but just reuse the processed tfrecords.

Thanks in advance!

**Comment by SoumyadeepJana** [comment]
@vishal-keshav Setting download=False in tfds.load() won't download the dataset again but will reuse the existing TFrecords

**Comment by vishal-keshav** [comment]
@SoumyadeepJana Thanks for the input. However, it did not solve my issue.

Here is the explanation.
I have been using the above mentioned [download_and_convert_imagenet.sh](https://github.com/tensorflow/models/blob/master/research/slim/datasets/download_and_convert_imagenet.sh) script for downloading and building tfrecords for train and validation of ImageNet datasets.

If you see closely, the number of tfrecord files for train datasets are 1024 while the number of tfrecord files for validations dataset are 128 as created by the imagenet script.

Now, here was the problem.
I assumed that tensorflow_datasets `download_and_prepare()` API also pre-process and creates the same tfrecords as mentioned above. This was a wrong assumption.

I have to download the two files namely `ILSVRC2012_img_train.tar` and `ILSVRC2012_img_val.tar` again. The `download_and_prepare()` API created the tfrecords that are different than that of created by `download_and_convert_imagenet.sh`. Specifically, the tensorflow_datasets API created 1000 train tfrecords and 5 validation tfrecords. Hence, I cannot reuse the tfrecords created by `download_and_convert_imagenet.sh`.

In summary, there is no relation between how imagenet is preprocessed by the two mechanism mentioned above.

Closing the issue. Thank you.
